src/main/java/g0301_0400/s0349_intersection_of_two_arrays/Solution.java

```diff
@@ -0,0 +1,23 @@
+package g0301_0400.s0349_intersection_of_two_arrays;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// #Easy #Array #Hash_Table #Sorting #Binary_Search #Two_Pointers
+
+public class Solution {
+    public int[] intersection(int[] nums1, int[] nums2) {
+        List<Integer> intersectionList = new ArrayList<>();
+
+        for (int j : nums1) {
+            for (int k : nums2) {
```

Comment:
Please use faster solution this is too slow.

![image](https://user-images.githubusercontent.com/348496/145668395-2a1e1f5d-5ac7-489c-8503-6ef68477e98f.png)